docs/advanced-config.md

```diff
@@ -0,0 +1,104 @@
+#Advanced Configuration
+
+There are many ways to add Netlify CMS to your static site. This is by no means a comprehensive guide, but some of the more popular use-cases will be covered, including:
+
+* Configuration for a self-hosted Netlify CMS.
+* Configuring a third-party Github Oauth provider.
+* Using Github Enterprise repositories.
+
+If there's any other use-cases you're interested in or information you think is missing, you can file a [feature request](https://github.com/netlify/netlify-cms/issues), or [help us add it](https://github.com/netlify/netlify-cms/blob/master/CONTRIBUTING.md). Let's get started!
+
+## Register a Github Oauth App
+Information is available on the [Github Developer Documentation](https://developer.github.com/apps/building-integrations/setting-up-and-registering-oauth-apps/registering-oauth-apps/). Fill out the fields however you like, however **it is critical that your _authorization callback URL_ is correct**. This tells Github where to send a user after they've authenticated, and should match the URL of your Oauth provider. Use `https://your.app.com/callback` with Vencax's Oauth provider explained below.
```

Comment:
We'll want to avoid centering project documentation around individual contributors (in this case Vencax). His (stellar!) contribution is the only one currently, but we're hoping to see more. A direction mention of his repo should happen just once, and in some sort of list that can be added to. Would you mind making that adjustment?